libretro-common/streams/interface_stream.c

```diff
@@ -68,12 +69,12 @@ int64_t intfstream_get_size(intfstream_internal_t *intf)
          return filestream_get_size(intf->file.fp);
       case INTFSTREAM_MEMORY:
          return intf->memory.buf.size;
-      case INTFSTREAM_CHD:
 #ifdef HAVE_CHD
-        return chdstream_get_size(intf->chd.fp);
-#else
-        break;
+      case INTFSTREAM_CHD:
+         return chdstream_get_size(intf->chd.fp);
 #endif
+      default:
+         break;
```

Comment:
default cases added to all of these switches to prevent warnings when building with HAVE_CHD=0